cmd/provisioner-localpv/app/helper_hostpath.go

```diff
@@ -140,11 +145,12 @@ func (p *Provisioner) createCleanupPod(pOpts *HelperPodOptions) error {
 		return vErr
 	}
 
-	cPod, err := p.launchPod(config)
+	_, err := p.launchPod(config)
 	if err != nil {
 		return err
 	}
 
+	klog.Infof("Will not delete the helperPOD......")
 	if err := p.exitPod(cPod); err != nil {
```

Comment:
`cPod` is not defined in this scope. Can you try building the provisioner once.

Reply:
Done.